lib/iris/tests/integration/test_trajectory.py

```diff
@@ -203,7 +203,7 @@ def test_tri_polar__nearest(self):
         # TODO: arguably, we should support masked data properly in the
         # interpolation routine.  In the legacy code, that is unfortunately
         # just not the case.
-        test_cube.data.set_fill_value(0.0)
+        test_cube.fill_value = 0
```

Comment:
I'm just aping the same behaviour for this test, in terms of trajectories approach to dealing with masked data to yield the same expected result. Seems appropriate.